roles/openshift_default_storage_class/meta/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  author: Openshift Operations
+  description: This role configures the StorageClass in Openshift
+  company: Red Hat
+  license: Apache
+  min_ansible_version: 2.2
+  platforms:
+  - name: EL
+    verisons:
+    - 7
+  categories:
+  - cloud
```

Comment:
I think you need to add a dependency on lib_openshift here. like this, though without the other two dependencies

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_storage_glusterfs/meta/main.yml#L12-L15